xbmc/network/httprequesthandler/HTTPImageHandler.cpp

```diff
@@ -37,7 +38,13 @@ CHTTPImageHandler::CHTTPImageHandler(const HTTPRequest &request)
     XFILE::CImageFile imageFile;
     const CURL pathToUrl(file);
     if (imageFile.Exists(pathToUrl))
+    {
       responseStatus = MHD_HTTP_OK;
+      struct __stat64 buffer;
```

Comment:
Please see https://github.com/Rechi/xbmc/blob/92cd69039cc531a9b193e1636f1df480d6c76fea/xbmc/network/httprequesthandler/HTTPFileHandler.cpp#L90 for how to do this properly. Maybe we can refactor that logic into a helper method in `CHTTPFileHandler` which can be used by the base and any derived classes.